### DISCUSSION TITLE
(question) github-readme-stats api with theme parameters

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#1998] [Q&A] [answered]
Hey there!

**Warning: This is just a random idea that I had.***
*What if github-readme-stats has api with theme parameters like theme_name, title_color,icon_color, text_color, bg_color, border_color? would it be possible?*

#### 1. API
```
https://github-readme-stats.vercel.app/api?username=codehangen&
show_icons=boolean&
theme_name='value'&
title_color="value"&
icon_color="value"&
text_color="value"&
bg_color="value"&
border_color="value"
```

#### 2. Concept
```markdown
https://github-readme-stats.vercel.app/api?username=codehangen&show_icons=true&theme_name='name'&title_color="2f80ed"&icon_color="4c71f2"&text_color="434d58"&bg_color="fffefe"&border_color="e4e2e2"
```

#### 3. Example
```json

{

 "show_icons":"true",
 "theme_name":"name",
 "title_color":"2f80ed",
 "icon_color":"4c71f2",
 "text_color":"434d58",
 "bg_color":"fffefe",
 "border_color":"e4e2e2"
} 

```

#### 4. Sample
![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=codehangen&show_icons=true&theme=radical)

**Comment by rickstaa** [comment, marked as answer]
@codehangen Thanks for your feature suggestion. Not sure if I understand your suggestion, but to my knowledge, we already support those options. If you want to interact with the GRS App using a JSON API, a feature request can be found at https://github.com/anuraghazra/github-readme-stats/issues/624.

**&theme**

```md
![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&theme=radical)
```

![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&theme=radical)

**Other theme parameters**

```md
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&bg_color=30,e96443,904e95&title_color=fff&text_color=fff&border_color=ff0000)
```

 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&bg_color=30,e96443,904e95&title_color=fff&text_color=fff&border_color=ff0000)

**Reply by ghost** [reply]
Hi rickstaa.
> If you want to interact with the GRS App using a JSON API, a feature request can be found at https://github.com/anuraghazra/github-readme-stats/issues/624.

Exactly, what I want. thank you for feedback

**Comment by Marintania6** [comment]
#

**Reply by Janaya101** [reply]
what
